src/utils/ui/theme/wow-components.css

```
@@ -0,0 +1,523 @@
+  pointer-events: none;
+}
+
+/* Animations */
+@keyframes rain {
+  0%   { transform: translateY(-100vh); opacity: 0; }
+  10%  { opacity: 1; }
+  90%  { opacity: 1; }
+  100% { transform: translateY(100vh); opacity: 0; }
+}
+
+@keyframes voiceBar {
```

Stylelint (17.6.0)

[error] Expected keyframe name "voiceBar" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)